packages/teams-js/README.md

```diff
@@ -1,6 +1,6 @@
 # Microsoft Teams JavaScript client SDK
 
-Welcome to the Teams JavaScript client SDK ! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch.
+Welcome to the Teams JavaScript client SDK! For breaking changes, please refer to our changelog in the current `<root>/packages/teams-js` directory of the `2.0-preview` branch.
```

Comment:
I think it could be clearer to link directly to the CHANGELOG file, e.g.:

"For breaking changes, please refer to our [CHANGELOG](./CHANGELOG.md) in the current `<root>/packages/teams-js` directory of the `2.0-preview` branch."

(Note: I didn't validate that link works in my example)

Reply:
Great idea, I've linked it in my local commit and will push soon :)

Reply:
Updated

Reply:
I don't actually see the link in the code here. It's okay if you decided not to add the link, but if you did, it's not showing for me.

Reply:
Very weird. It's also not showing for me in the GitHub UI but it's there in my VS Code. 
<img width="775" alt="image" src="https://user-images.githubusercontent.com/69540388/154384708-ddc129ed-7957-4de2-a726-10b2d694b52f.png">
Could you do me a favor and see what it looks like if you pull my branch?

Reply:
Wait.... I have the wrong README. lol I'm going to fix it.